### PR TITLE
adding the book link function and organizing a few other things

### PR DESCRIPTION
**Pull request by Doriaph**
I added the book_ling function, as well as book_name.
I also changes some names of attributes.

### CODE REVIEW

File: quote.py [hunk 1]
```diff
@@ -1,45 +1,54 @@
 class Quote:
     def __init__(self, quote):
         self.quote = quote
```
Comment (lernerbruno) [inline quote.py]:
```suggestion
        self.quote_html = quote
```

File: quote.py [hunk 2]
```diff
@@ -1,45 +1,54 @@
...
 
-    def __get_author(self, quote):
-        """
-        Extracts info about author from quote html element
-        """
+    def _get_content(self, quote):
+        """Takes an individualized raw quote and pulls out all of its
+        content as a string."""
+        content = quote.find('div', class_="quoteText").contents[0]
+        content = content.strip().strip(" ”“")
```
Comment (lernerbruno) [inline quote.py]: I would put this `" ”“" ` as a constant with a descriptive name

File: scraper.py [hunk 3]
```diff
@@ -12,10 +12,8 @@ def __init__(self, root_url):
                                   'html.parser')
 
     def __get_page_content(self, url):
```
Comment (ykestenberg) [inline scraper.py]:
```suggestion
    def _get_page_content(self, url):
```

File: scraper.py [hunk 4]
```diff
@@ -26,33 +24,30 @@ def __get_quotes_elements(self):
         return quotes
 
     def __create_quotes_objects(self, quotes):
```
Comment (ykestenberg) [inline scraper.py]:
```suggestion
    def _create_quotes_objects(self, quotes):
```

File: scraper.py [hunk 5]
```diff
@@ -26,33 +24,30 @@ def __get_quotes_elements(self):
...
             url_new = self.new_url(url, i)
             url_list.append(url_new)
         return url_list
 
     def scrap(self):
-        """
-        This function holds the scraping workflow
-        """
+        """This function holds the scraping workflow."""
         quotes = self.__get_quotes_elements()
```
Comment (ykestenberg) [inline scraper.py]:
```suggestion
        quotes = self._get_quotes_elements()
```

File: quote.py [hunk 6]
```diff
@@ -1,45 +1,54 @@
...
+        else:
+            return book_html.text
+
+    def _get_book_link(self, quote):
+        book_html = quote.find('a', class_="authorOrTitle")
+        if book_html is None:
+            return 'No book mentioned'
+        else:
+            uri = book_html['href']
+            host = 'https://www.goodreads.com'
```
Comment (lernerbruno) [inline quote.py]: I would create a constant for the host also

File: scraper.py [hunk 7]
```diff
@@ -26,33 +24,30 @@ def __get_quotes_elements(self):
...
         info = self.__create_quotes_objects(quotes)
```
Comment (ykestenberg) [inline scraper.py]:
```suggestion
        info = self._create_quotes_objects(quotes)
```

File: scraper.py [hunk 8]
```diff
@@ -26,33 +24,30 @@ def __get_quotes_elements(self):
...
         quotes_objects = []
         for q in quotes:
             new_quote = quote.Quote(q)
             quotes_objects.append(new_quote)
 
     def __new_url(self, url, page_num):
         """ Creates a proper URL containing the page number from the input. """
-        addition = '?page='
-        url_new = url + addition + str(page_num)
+        page_indicator = '?page='
```
Comment (lernerbruno) [inline scraper.py]: constant :)

File: quote.py (hunk 2 above)
Comment (Doriaph) [reply quote.py]: ok, will do that

File: quote.py (hunk 6 above)
Comment (Doriaph) [reply quote.py]: will do that also

File: scraper.py (hunk 8 above)
Comment (Doriaph) [reply scraper.py]: ok

**Pull request merged**